src/components/Book.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] I would recommend using `prop-types` library here instead of disabling the linters. PropTypes exports a range of validators that can be used to make sure the data you receive is valid. You can check [here](https://www.npmjs.com/package/prop-types) as well.

Reply:
Thank you for suggesting what to do but I deleted all the eslint disable and it is not showing any error.